geomstats/geometry/discrete_surfaces.py

```diff
@@ -514,26 +514,23 @@ def __init__(self, space, a0=1.0, a1=1.0, b1=1.0, c1=1.0, d1=1.0, a2=1.0):
         self.d1 = d1
         self.a2 = a2
 
-        self.exp_solver = DiscreteSurfacesExpSolver(space, n_steps=10)
+        if not gs.__name__.endswith("numpy"):
+            self.exp_solver = DiscreteSurfacesExpSolver(space, n_steps=10)
 
-        optimizer = ScipyMinimize(
-            method="L-BFGS-B",
-            jac="autodiff",
-            options={"disp": False, "ftol": 0.001},
-        )
-        self.log_solver = PathStraightening(space, n_nodes=10, optimizer=optimizer)
+            optimizer = ScipyMinimize(
+                method="L-BFGS-B",
+                jac="autodiff",
+                options={"disp": False, "ftol": 0.001},
+            )
+            self.log_solver = PathStraightening(space, n_nodes=10, optimizer=optimizer)
 
     def _inner_product_a0(self, tangent_vec_a, tangent_vec_b, vertex_areas_bp):
         r"""Compute term of order 0 within the inner-product.
 
         Denote h and k the tangent vectors a and b respectively.
-        Denote q the base point, i.e. the surface.
-
-        The equation of the inner-product is:
-        :math:`\int_M (G_{a_0} + G_{a_1} + G_{b_1} + G_{c_1} + G_{d_1} + G_{a_2})vol_q`.
+        Denote q the base point.
```

Comment:
Why did you remove the rest of this docstring? I felt that the additional explanations were helping.

Reply:
Ah yes, all good, let's keep it like this